functionMap.php

```diff
@@ -128,7 +128,7 @@
     'WP_Filesystem_ftpsockets::dirlist' => [$filesystemDirlistReturnType],
     'wpdb::prepare' => [null, 'query' => 'literal-string'],
     'wpdb::get_row' => ["null|void|(\$output is 'ARRAY_A' ? array<string, mixed> : (\$output is 'ARRAY_N' ? array<int, mixed> : \stdClass))", 'output' => "'OBJECT'|'ARRAY_A'|'ARRAY_N'", 'y' => '0|positive-int'],
-    'wpdb::get_results' => ["null|(\$output is 'ARRAY_A' ? array<string, mixed> : (\$output is 'ARRAY_N' ? array<int, mixed> : (\$output is 'OBJECT_K' ? array<string, \stdClass> : \stdClass)))", 'output' => "'OBJECT'|'OBJECT_K'|'ARRAY_A'|'ARRAY_N'"],
+    'wpdb::get_results' => ["null|(\$output is 'ARRAY_A' ? list<array<string, mixed>> : (\$output is 'ARRAY_N' ? list<array<int, mixed>> : (\$output is 'OBJECT_K' ? array<array-key, \stdClass> : list<\stdClass>)))", 'output' => "'OBJECT'|'OBJECT_K'|'ARRAY_A'|'ARRAY_N'"],
```

Comment:
```suggestion
    'wpdb::get_results' => ["null|(\$output is 'ARRAY_A' ? list<array<string, mixed>> : (\$output is 'ARRAY_N' ? list<array<int, mixed>> : (\$output is 'OBJECT_K' ? list<array<array-key, \stdClass>> : list<\stdClass>)))", 'output' => "'OBJECT'|'OBJECT_K'|'ARRAY_A'|'ARRAY_N'"],
```

```php
```php
global $wpdb;
$query1 = "
	SELECT ID, post_title
	FROM $wpdb->posts
	WHERE ID = 85229
";
$query2 = "
	SELECT post_title
	FROM $wpdb->posts
	WHERE ID = 85229
";
var_dump($wpdb->get_results($query1, 'OBJECT_K'));
// output:
// array(1) { [85229]=> object(stdClass)#16115 (2) { ["ID"]=> string(5) "85229" ["post_title"]=> string(8) "Comments" } }
var_dump($wpdb->get_results($query2, 'OBJECT_K')); 
// output:
// array(1) { ["Comments"]=> object(stdClass)#16115 (1) { ["post_title"]=> string(8) "Comments" } }
```